src/web/pipeline/server.nginx.conf

```
@@ -67,6 +80,10 @@ server {
 			proxy_pass http://$1/$1$2$is_args$args;
 		}
 
+		location /api/s/ {
+			proxy_pass http://space-proxy;
```

Давай так попробуем избавится от лишнего regex group в 119 строке 
```suggestion
			proxy_pass http://space-proxy/;
```

---

Получилось.